### ISSUE TITLE
Setting random seeds

### ISSUE THREAD

**Issue by alrichardbollans** [dsilvestro/npBNN#3]
In a few of the bn functions (e.g. `get_data`, `npBNN.init`) the global seed is being set --- `np.random.seed(seed)`. This is causing issues for me with other processes that use randoms numbers. In my case, this is affecting the generation of train/test indices in cross validation. If running the following example, the train/test indices are simply repeated where they shouldn't be. I believe the fix to this is to use a Random object and set its seed instead of the global seed.

```
    import np_bnn as bn
    import pandas as pd
    from sklearn.model_selection import RepeatedKFold

    # Example data
    from import_trait_data import NUMERIC_TRAITS, IMPORTED_TRAIT_CSV
    train_data = pd.read_csv(IMPORTED_TRAIT_CSV)
    X = train_data[NUMERIC_TRAITS]
    y = train_data['Activity_Antimalarial']

    rskf = RepeatedKFold(n_splits=2, n_repeats=5)

    for i, (train_index, test_index) in enumerate(rskf.split(X, y)):
        print(f'Fold: {i}')
        print(train_index)
        print(test_index)
        y_train, y_test = y.iloc[train_index], y.iloc[test_index]
        X_train, X_test = X.iloc[train_index], X.iloc[test_index]

        train_dat = bn.get_data(X_train, y_train,
                                seed=1234,
                                testsize=0,
                                randomize_order=False,
                                instance_id=None, from_file=False)
```

**Comment by dsilvestro** [comment]
Hi Adam thanks for raising the issue - I think this is now fixed wit the latest version I just pushed!
